### PR TITLE
add via support for dztech 60 and 65 pcb

### PR DESCRIPTION
**Pull request by moyi4681**

### CODE REVIEW

File: keyboards/dztech/dz60rgb_wkl/dz60rgb_wkl.h [hunk 1]
```diff
@@ -8,14 +8,12 @@
...
+    K40, K41, K42,           K45,                     K4A, K4B,      K4D  \
 ) { \
     { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D }, \
     { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D }, \
     { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2C, K2D }, \
     { K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, XXX, K3D }, \
     { K40, K41, K42, XXX, XXX, K45, XXX, XXX, XXX, XXX, K4A, K4B, XXX, K4D }  \
 }
-
-#define LAYOUT_HHKB LAYOUT_60_tsangan_hhkb
```
Comment (zvecr) [inline keyboards/dztech/dz60rgb_wkl/dz60rgb_wkl.h]: This was added for backwards compatibility, and should remain till a breaking changes window.

File: keyboards/dztech/dz60rgb/dz60rgb.h [hunk 2]
```diff
@@ -8,8 +8,8 @@
     K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, \
     K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, \
     K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B,      K2D, \
-    K30,      K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3D, \
-    K40, K41, K42,                K45,           K48, K49, K4A, K4B, K4D  \
+    K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B,      K3D, \
+    K40, K41, K42,           K45,           K48, K49, K4A, K4B,      K4D  \
```
Comment (fauxpark) [inline keyboards/dztech/dz60rgb/dz60rgb.h]:
```suggestion
    K30,      K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3D, \
    K40, K41, K42,                K45,           K48, K49, K4A, K4B, K4D  \
```
These are correct.

File: keyboards/dztech/dz60rgb_ansi/dz60rgb_ansi.h [hunk 3]
```diff
@@ -8,8 +8,8 @@
     K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, \
     K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, \
     K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B,      K2D, \
-    K30,      K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A,      K3B, \
-    K40, K41, K42,                K45,                K49, K4A, K4B, K4D  \
+    K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B,           \
+    K40, K41, K42,           K45,                K49, K4A, K4B,      K4D  \
```
Comment (fauxpark) [inline keyboards/dztech/dz60rgb_ansi/dz60rgb_ansi.h]:
```suggestion
    K30,      K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A,      K3B, \
    K40, K41, K42,                K45,                K49, K4A, K4B, K4D  \
```

File: keyboards/dztech/dz60rgb_wkl/dz60rgb_wkl.h [hunk 4]
```diff
@@ -8,14 +8,12 @@
     K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K2C, \
     K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, \
     K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B,      K2D, \
-    K30,      K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3D, \
-    K40, K41, K42,                K45,                     K4A, K4B, K4D  \
+    K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B,      K3D, \
+    K40, K41, K42,           K45,                     K4A, K4B,      K4D  \
```
Comment (fauxpark) [inline keyboards/dztech/dz60rgb_wkl/dz60rgb_wkl.h]:
```suggestion
    K30,      K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3D, \
    K40, K41, K42,                K45,                     K4A, K4B, K4D  \
```

File: keyboards/dztech/dz65rgb/dz65rgb.h [hunk 5]
```diff
@@ -8,8 +8,8 @@
     K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E, \
     K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, K1E, \
     K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B,      K2D, K2E, \
-    K30,      K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3D, K3E, \
-    K40, K41, K42,                K45,           K48, K49, K4A, K4B, K4D, K4E  \
+    K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B,      K3D, K3E, \
+    K40, K41, K42,           K45,           K48, K49, K4A, K4B,      K4D, K4E  \
```
Comment (fauxpark) [inline keyboards/dztech/dz65rgb/dz65rgb.h]:
```suggestion
    K30,      K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3D, K3E, \
    K40, K41, K42,                K45,           K48, K49, K4A, K4B, K4D, K4E  \
```

File: keyboards/dztech/dz60rgb_wkl/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        KC_MPRV,          KC_VOLD, KC_VOLU, KC_MUTE, _______, _______, NK_TOGG, _______, _______, KC_END,  KC_PGDN, KC_MNXT, _______,
+        _______, _______, _______,                            _______,                                     _______, _______, _______
+    ),
+    [2] = LAYOUT_60_tsangan_hhkb(
+        _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,  _______,
+        _______, RGB_TOG, _______, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, RGB_MOD, _______, _______, _______, RESET,
+        _______, _______, _______, _______, _______, _______, _______, _______, RGB_SPI, RGB_SPD, _______, _______,          _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______,                            _______,                                     _______, _______, _______
+    )
```
Comment (noroadsleft) [inline keyboards/dztech/dz60rgb_wkl/keymaps/via/keymap.c]: VIA support defaults to four layers.

```suggestion
    ),
    [3] = LAYOUT_60_tsangan_hhkb(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______,                            _______,                                     _______, _______, _______
    )
```

File: keyboards/dztech/dz65rgb/keymaps/via/keymap.c [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+        KC_LSFT,                 KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
+        KC_LCTL,        KC_LGUI, KC_LALT,                            KC_SPC,                    KC_RALT, MO(1),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT_65_ansi(
+        KC_GESC,        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,  KC_HOME,
+        _______,        RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, _______, KC_PSCR, KC_SLCK, KC_PAUS, RESET,   KC_PGUP,
+        CTL_T(KC_CAPS), RGB_SPI, RGB_SPD, _______, _______, _______, _______, _______, _______, _______, _______, _______,          EEP_RST, KC_PGDN,
+        KC_LSFT,                 _______, _______, _______, _______, _______, NK_TOGG, _______, _______, _______, _______, _______, KC_VOLU, KC_MUTE,
+        _______,        _______, _______,                            _______,                   _______, _______, _______, KC_MPRV, KC_VOLD, KC_MNXT
+    )
```
Comment (noroadsleft) [inline keyboards/dztech/dz65rgb/keymaps/via/keymap.c]:
```suggestion
    ),
    [2] = LAYOUT_65_ansi(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______,                            _______,                   _______, _______, _______, _______, _______, _______
    ),
    [3] = LAYOUT_65_ansi(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______,                            _______,                   _______, _______, _______, _______, _______, _______
    )
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**